tests/__mocks__/quick-lru.ts

```
@@ -0,0 +1,56 @@
+      this.keyOrder.splice(index, 1)
+    }
+    return super.delete(key)
+  }
+
+  clear(): void {
+    this.keyOrder.length = 0
+    super.clear()
+  }
+}
+
+export default MockQuickLRU 
```

ESLint

[error] Trailing spaces not allowed.

(@stylistic/no-trailing-spaces)